src/context/concrete_base.rs

```diff
@@ -17,6 +17,34 @@ use crossbeam_channel::{Receiver, SendError, Sender};
 use std::sync::atomic::{AtomicU64, AtomicU8, Ordering};
 use std::sync::{Arc, Mutex, RwLock, RwLockWriteGuard};
 
+/// This struct assigns new [`AudioNodeId`]s for [`AudioNode`]s
+///
+/// It reuses the ids of decommissioned nodes to prevent unbounded growth of the audio graphs node
+/// list (which is stored in a Vec indexed by the AudioNodeId).
+struct AudioNodeIdIssuer {
```

Comment:
maybe rename to `AudioNodeIdProvider`? 
I find `audio_node_id_issuer.issue()` a bit weird (maybe `audio_node_id_provider.get()`? But really no big deal